internal/mining/mining_harness_test.go

```diff
@@ -678,7 +682,7 @@ func (p *fakeTxSource) maybeAcceptTransaction(tx *dcrutil.Tx, isNew bool) ([]*ch
 					txIn.PreviousOutPoint.Hash)
 				continue
 			}
-			if entry.IsFullySpent() {
+			if entry.IsSpent() {
 				log.Tracef("Transaction %v uses full spent input %v "+
```

Comment:
s / ... uses full spent input ... / ... uses spent input ...

Reply:
Good call, fixed this case and a similar case in `mempool_test.go`.